### ISSUE TITLE
Package conflicts with mesos

### ISSUE THREAD

**Issue by stevendborrelli** [CiscoCloud/microservices-infrastructure#1025]
If the mesosphere repo is enabled ( and it is in default mantl) a yum upgrade might cause the following conflicts:

```
Finished Dependency Resolution
Error: Package: mesos-master-0.25.0-1.noarch (@/mesos-master-0.25.0-1.noarch)
           Requires: mesos = 0.25.0
           Removing: mesos-0.25.0-1.x86_64 (@/mesos-0.25.0-1.x86_64)
               mesos = 0.25.0-1
           Updated By: mesos-0.26.0-0.2.145.centos701406.x86_64 (mesosphere)
               mesos = 0.26.0-0.2.145.centos701406
```

Could we remove the remaining mesosphere packages and/or make sure our package is not causing this conflicts. 

1. Zookeeper (we can package this)
2. Chronos (this should move to a framework, it does not work well in HA mode anyway)

Pinging @avnik and @BrianHicks

**Comment by avnik** [comment]
@stevendborrelli  which action produce this conflict?

**Comment by stevendborrelli** [comment]
`yum upgrade` on a provisioned cluster

**Comment by avnik** [comment]
Posting them a ticket  also can be solution.
If I can simulate this problem, I can try to fix mesosphere's packages as well

**Comment by avnik** [comment]
Are upgrade conflict happens, when system try to replace mesosphere's packages with our own?

**Comment by ryane** [comment]
Tracking this for 0.6. As I understand it, a potential short term solution is to release a new iteration of the mesos package that includes an `obsoletes` line.

A workaround to upgrade everything, except zk and chronos, on existing clusters: `yum upgrade --disablerepo=mesosphere,mesosphere-noarch`

**Comment by ryane** [comment]
fixed by #1092
